docs-source/spring/content/development/cli/_index.md

```diff
@@ -957,6 +1011,208 @@ GraalVM Compile Commands
             E-mail: obaas_ww@oracle.com
     ```
 
+### User Management
```

Comment:
You may want incorporate this information section User Management REST endpoints from https://oracle.github.io/microservices-datadriven/spring/security/azn-server/#user-management-rest-endpoints so the user knows what rights are required for the commands